src/app/modules/form/admin-form/admin-form.utils.ts

```diff
@@ -57,6 +58,11 @@ export const mapRouteError = (
   coreErrorMessage?: string,
 ): ErrorResponseData => {
   switch (error.constructor) {
+    case SmsLimitExceededError:
+      return {
+        statusCode: StatusCodes.CONFLICT,
```

Comment:
rethinking this status code. Would returning 400 be better?

Reply:
to give context, this error is thrown when sms limit is exceeded but the UI does not reflect it yet; hence, i thought `409 conflict` would be better as it represents a [difference](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409) between the UI state and BE state. 

could i understand why you think 400 is better?

Reply:
> represents a difference between the UI state and BE state.

This is not a particularly correct justification; all PUT requests inherently have a difference between client and BE before the request is resolved (that's the whole point of a PUT, to sync backend and client state).

But yea, no big deal, 409 works too